libraries/chain/genesis.cpp

```diff
@@ -10,18 +10,18 @@
 
 #include <scorum/chain/pool/reward_pool.hpp>
 
+#include <scorum/chain/genesis.hpp>
+
 #include <fc/io/json.hpp>
 
-#define SCORUM_DEFAULT_INIT_PUBLIC_KEY "STM5omawYzkrPdcEEcFiwLdEu7a3znoJDSmerNgf96J2zaHZMTpWs"
+#define SCORUM_DEFAULT_INIT_PUBLIC_KEY "SCR5omawYzkrPdcEEcFiwLdEu7a3znoJDSmerNgf96J2zaHZMTpWs"
```

Comment:
It is not necessary to create account "initdelegate" in generate_default_genesis_state 
(and public key without private key). Everyone need full json to run node in real world